app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,17 @@
+from .dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(self, nickname: str,
+                 favourite_dish: str,
+                 hummer_level: int) -> None:

```

Comment:
There is a typo in the attribute name `hummer_level`. It should be corrected to `hammer_level`.